diagnostic.md

```diff
@@ -76,7 +76,7 @@ var groceryList = [
 
 What is the value of `groceryList[2]`?
 
-_Replace this text with your answer._
+'a container of milk'
```

Comment:
Array indexes start with 0, which makes index point to the 3rd object in the array, 'a stick of butter'.